src/CoreBundle/DependencyInjection/Compiler/FormCompilerPass.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @deprecated since sonata-project/core-bundle 3.19, to be removed in 4.0.
```

Comment:
```suggestion
 * @deprecated since sonata-project/core-bundle 3.20, to be removed in 4.0.
```
Or
```suggestion
 * @deprecated since sonata-project/core-bundle 3.x, to be removed in 4.0.
```
And we'll replace `3.x` by the right release number.

Same for others.